demo/index.html

```diff
@@ -0,0 +1,130 @@
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <title>paper-button</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link href="../../font-roboto/roboto.html" rel="import">
+  <link href="../../core-icons/core-icons.html" rel="import">
+  <link href="../paper-button.html" rel="import">
+  <style>
+    body {
+      font-family: RobotoDraft, 'Helvetica Neue', Helvetica, Arial;
+      font-size: 14px;
+      padding: 24px;
+      margin: 0;
+    }
+
+    section {
+      display: block;
+      position: relative;
+      padding: 20px 0;
+    }
+
+    section > div {
+      padding: 14px;
+      font-size: 16px;
+    }
+
+    paper-button.colorful {
+      color: #4285f4;
+    }
+
+    paper-button[raised].colorful {
+      background: #4285f4;
+      color: #fff;
+    }
+
+    paper-button[toggle] {
+      transition: background-color 0.3s;
+    }
+
+    paper-button[toggle][active] {
+      background-color: rgba(0, 0, 0, 0.25);
+    }
+
+    paper-button[toggle][active].colorful {
+      background-color: rgba(66, 133, 244, 0.25);
+    }
+
+    paper-button[toggle][active][raised].colorful {
+      background-color: rgba(66, 133, 244, 0.75);
+    }
+
+    paper-button.hover:hover {
+      background: #eee;
+    }
+
+    paper-button.blue-ripple::shadow paper-ripple {
+      color: #4285f4;
+    }
+
+    paper-button.blue-ripple paper-ripple {
+      color: #4285f4;
+    }
+  </style>
+</head>
+<body>
+
+  <h1>Paper Buttons</h1>
+  <h2>Flat</h2>
+
+  <paper-button tabindex="0">button</paper-button>
+  <paper-button tabindex="0" class="colorful">colorful</paper-button>
+  <paper-button tabindex="0" disabled>disabled</paper-button>
+  <paper-button tabindex="0" noink>noink</paper-button>
+
+  <h2>Raised</h2>
+
+  <paper-button tabindex="0" raised>button</paper-button>
+  <paper-button tabindex="0" raised class="colorful">colorful</paper-button>
+  <paper-button tabindex="0" raised disabled>disabled</paper-button>
+  <paper-button tabindex="0" raised noink>noink</paper-button>
+
+  <h2>Custom Content</h2>
+  <paper-button tabindex="0" class="colorful custom">
+    <core-icon icon="check"></core-icon>
+    ok
+  </paper-button>
+
+  <paper-button tabindex="0" class="custom">
+    <core-icon icon="clear"></core-icon>
+    cancel
+  </paper-button>
+
+  <paper-button tabindex="0">
+    <a href="https://www.polymer-project.org" target="_blank">link</a>
```

Comment:
Nuke the anchor examples? There could be a `paper-button-anchor` (name is not important) that implements `<a is="paper-button-anchor"></a>`.